### DISCUSSION TITLE
[Question] How to just serve one view?

### DISCUSSION THREAD

**Discussion by mpikmwm** [flet-dev/flet#1362] [Q&A]
### Question

Hey,
I want every user to see the same view. This is because I intent to build an app that can start and stop measurements. But right now I can start multible measurements and I want to prevent this to happen. Also all useres should be able to see the same measurement results user A made.

So basicly all useres see the exact same page. Is this possible?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Store application state in a singleton class which can be displayed/controlled by multiple user sessions.
